ironfish/src/network/peers/connections/webRtcConnection.ts

```diff
@@ -193,15 +196,18 @@ export class WebRtcConnection extends Connection {
       return false
     }
 
-    if (!this.datachannel.isOpen()) {
-      this.logger.debug('Datachannel no longer open, closing connection')
-      this.close()
-      return false
+    let sent = false
```

Comment:
Curious was there another error you ran into here beside the channel being closed?

Reply:
I think we don't actually need this now, I'll try to remove it. I started tinkering around with the code when I found that you can't mock functions in ES Module packages, and we were mocking the implementation of `isOpen`. But I think the `connection` class already handles try/catching errors and logging them, so we probably don't need either the `try/catch` here, or `isOpen`.

Reply:
Ah yep you're right, good call

Reply:
All right, I rolled this back, but I did leave the change that uses the return value of `sendMessageBinary`, since I think we should be handling that value instead of ignoring it.